actionview/app/assets/javascripts/rails-ujs/utils/event.coffee

```diff
@@ -66,3 +66,4 @@ Rails.delegate = (element, selector, eventType, handler) ->
     if target instanceof Element and handler.call(target, e) == false
       e.preventDefault()
       e.stopPropagation()
+  , true
```

Comment:
this line makes the tests fail - needs to be removed before merging, but i want to see a CI run with failures to confirm it works